tests/unittest_builder.py

```diff
@@ -78,9 +79,12 @@ def test_callfunc_lineno(self) -> None:
         self.assertEqual(name.tolineno, 4)
         strarg = callfunc.args[0]
         self.assertIsInstance(strarg, nodes.Const)
-        if hasattr(sys, "pypy_version_info"):
+        if PYPY:
             self.assertEqual(strarg.fromlineno, 4)
-            self.assertEqual(strarg.tolineno, 4)
+            if not PY39_PLUS:
+                self.assertEqual(strarg.tolineno, 4)
+            else:
```

Comment:
The `else` is not "covered" right now, right?

Reply:
Correct. I tested PyPy 3.9 locally and this seems to be the only necessary change regarding the lineno and col_offset attributes. PyPy 3.9 is basically equivalent with CPython there.

_Unfortunately there is another issue that prevents us from adding PyPy 3.9 here, too._